build-libs/redirects.js

```diff
@@ -52,12 +53,28 @@ const HOSTNAME_MAP = {
  * @param {string} redirectsPath Path within the repo to the redirects file.
  * @returns {Promise<Redirect[]>}
  */
-async function getRedirectsFromContentRepo(repoName, redirectsPath) {
+async function getRedirectsFromContentRepo(repoName, redirectsPath, config) {
 	/**
 	 * Note: These constants are declared for clarity in build context intent.
 	 */
 	const isDeveloperBuild = !process.env.IS_CONTENT_PREVIEW
 	const isLocalContentBuild = isDeployPreview(repoName)
+	/**
+	 * Load redirects from the unified docs repo if it's in the list of migrated repos.
+	 * Return early if there are not any redirects found for that specific repo.
+	 */
+	if (
+		config.flags?.unified_docs_migrated_repos?.find((repo) => repo === repoName)
```

Comment:
@LeahMarieBush can you also add a check if we are in the using the unified-docs-sandbox?

process.env.HASHI_ENV === 'unified-docs-sandbox'